### PR TITLE
Added the ability to do github-slack username mappings

### PR DESCRIPTION
**Pull request by AntonNguyen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,6 +21,14 @@ separated by commas
 6. Enter in the channel you want the bot to slack into the Heroku config `SLACK_CHANNEL` field
 7. Click Deploy for Free
 
+Mapping Github Usernames To Slack Username
+============================================
+In the Heroku `TEAM_MEMBERS` config, for any username that needs a mapping, simply specify their name as <github_name>:<slack_name>. For example
+```
+TEAM_MEMBERS: anton,markstory:THEmarkstory,anandh
```
Comment (iamjillsanluis) [inline README.md]: Would be nice to make this the correct mapping 😄 . I am assuming you are making `anton` map to `THEmarkstory`?

Comment (iamjillsanluis) [inline README.md]: Would be nice to make this more of 'githubname':'slack','anothergithubname':'anotherslack',<etc>. I find the original approach error prone (in previous life). If we miss one entry, we have to squint ever so slightly to make sure the entries match. 😄

Comment (AntonNguyen) [inline README.md]: `,` is the name separator, and `:` is the mapping, so `markstory:THEmarkstory` is the mapping.

Comment (iamjillsanluis) [inline README.md]: yeah, but the example is not too clear on that - `anton,markstory:THEmarkstory,anandh` yields to `anton:THEmarkstory`, `markstory:anandh` unless you have invented a cloning device.

**Pull request merged**